content/python/concepts/strings/terms/join/join.md

```diff
@@ -0,0 +1,49 @@
+---
+Title: '.join()'
+Description: 'Concatenates all items from an iterable into a single string.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Strings'
+  - 'Methods'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The `.join()` method concatenates all items from an iterable into a single string.
+
+## Syntax
+
+```pseudo
+string.join(iterable)
+```
+
+- `string` is the separator `.join()` is called on. The separator is used to separate each element of the iterable. It can be any string including an empty string.
```

Comment:
```suggestion
- The `.join()` method is called on a separator of type string. The separator can be any string, including an empty string, and is used to separate each element of the iterable.
```

I think it's important to make some distinction that the `string` mentioned in the pseudo block represents any string. This is why I'm suggesting taking away the backticks.